src/pages/home/_trade-types.tsx

```diff
@@ -249,13 +272,15 @@ const TradeTypes = (): React.ReactNode => {
                 mb="40px"
                 tablet={{ mb: '16px' }}
             >
-                {localize('Trade the way you want with 3 exciting trade types')}
+                {is_row
+                    ? localize('Trade the way you want with 3 exciting trade types')
+                    : localize('Trade the way you want with 2 exciting trade types')}
```

Comment:
```suggestion
                <Localize translate_text="Trade the way you want with {{trade_no}} exciting trade types" values={{ trade_no: is_row ? '3' : '2' }} />
```

There's a better way to handle dynamic text translations as I shown above. You can refer to the `General Implementation Guide` for additional info